docs/_includes/confluence.adoc

```diff
@@ -0,0 +1,70 @@
+////
+Header: Convert Confluence XHTML to Asciidoctor
+
+Included in:
+
+- user-manual
+////
+
+You can convert Atlassian Confluence XHTML pages to Asciidoctor using this http://www.groovy-lang.org/download.html[Groovy] script.
+
+The script calls the http://pandoc.org/[Pandoc] tool to convert single or multiple HTML files exported from Confluence to AsciiDoc files (using the Python version of the AsciiDoc toolchain).
```

Comment:
> Does pandoc ship the "backends" with the tool?

pandoc does AsciiDoc in its own way. Totally custom. It doesn't use anything from the AsciiDoc or Asciidoctor ecosystems (which is part of the reason why I don't like the output it produces, but that's a rant for a different day).